src/Amadeus.Server/Views/User/UserView.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amadeus.Server.Controllers;
+using Amadeus.Server.Models;
+using Amadeus.Server.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Amadeus.Server.Views
+{
+	/// <summary>
+	/// User view that receives http requests.
+	/// </summary>
+	[ApiController]
+	[Route("/user")]
+	public class UserView : ControllerBase
+	{
+		private readonly UserController _userController;
+
+		/// <summary>
+		/// Create a new <see cref="UserView"/>.
+		/// </summary>
+		/// <param name="userController">Controller for the business logic.</param>
+		public UserView(UserController userController)
+		{
+			_userController = userController;
+		}
+
+		/// <summary>
+		/// Get all users.
+		/// </summary>
+		/// <remarks>
+		/// It gets all the users.
+		/// </remarks>
+		/// <returns>All the users.</returns>
+		[HttpGet]
+		[ProducesResponseType(StatusCodes.Status200OK)]
+		public async Task<ActionResult<List<Models.User>>> GetAll()
+		{
+			return await _userController.GetUsers();
+		}
+
+		/// <summary>
+		/// Get a specific user.
+		/// </summary>
+		/// <param name="uid">Uid of the specific user.</param>
+		/// <returns>All infos of the specific user.</returns>
+		[HttpGet("{uid:int}")]
+		public async Task<ActionResult<User>> GetUser(int uid)
+		{
+			return await _userController.GetUser(uid);
+		}
+
+		/// <summary>
+		/// Create a user.
+		/// </summary>
+		/// <param name="user">The user to create.</param>
+		/// <returns>The infos of the newly created user.</returns>
+		[HttpPost]
+		public async Task<IActionResult> CreateUser(User user)
+		{
+			try
+			{
+				await _userController.CreateUser(user);
+			}
+			catch (ArgumentException exception)
+			{
+				return BadRequest();
+			}
+			return CreatedAtAction(nameof(CreateUser), new { uid = 42 }, user);
```

Comment:
You should use the Created attribute and return the user that your repository returned.